vtex/resources-manager-ui
# Feature/product resources page

#### What does this PR do? \*
Creates a page to list resources from a product. This page will redirect to resource edition and resource creation page.

#### How to test it? \*
Go to https://psmeirelesproductresources--storecomponents.myvtex.com/admin/resources-manager/. Select a product, search for some resources, click in any row and see the url being modified, as well as when clicking to add new resource. Button to go back should work.

[ch28979]
